src/helpers/removeTrailingNewLine.js

```diff
@@ -24,10 +24,10 @@ const removeTrailingNewLine = (pdf) => {
 
     const lastLine = output.slice(output.length - 6).toString();
     if (lastLine !== '\n%%EOF') {
-        throw new SignPdfError(
-            'A PDF file must end with an EOF line.',
-            SignPdfError.TYPE_PARSE,
-        );
+        output = Buffer.concat([
```

Comment:
Not quite sure about that. We don't want to alter the document by appending EOF when one is not present. Or do we?